### DISCUSSION TITLE
Compatibility with old Mac OSX

### DISCUSSION THREAD

**Discussion by Nikohaku** [christofmuc/KnobKraft-orm#331] [General]
Is there a version compatible with Mac OS 10.13.6 ?

**Comment by christofmuc** [comment]
Theoretically, old Macs were supported for quite a while. I am not entirely sure on the conditions required, your best bet would be to try different releases back in time and see if you can get any to launch. T

You can try this version, it was explicitly built for old MacOS versions:
https://github.com/christofmuc/KnobKraft-orm/releases/tag/1.6.3

I can't remember the outcome, let's have a look at the error messages when you try to run 1.6.3.

**Comment by christofmuc** [comment]
Looking at this again, I fear there is a lot of work required to get the old MacOS going, I am not sure this is worth the effort given that OS X 10.13 High Sierra hasn't received updates for near 4 years now.
